modules/custom/cu_rss/cu_rss.module

```diff
@@ -292,12 +292,10 @@ function cu_rss_secure_permissions($role) {
       'administer rss feeds',
     ),
     'edit_my_content' => array(),
-
-    'edit_only' => array(),
-
     'content_editor' => array(
       'administer rss feeds',
     ),
+    'edit_only' => array(),
```

Comment:
I don't think an empty array for each role is needed due to the `isset()` check at the end of the function.